src/VM/OpcodeHandler.h

```diff
@@ -27,6 +27,15 @@
 
 // Third party includes
 
+#if defined(_MSC_VER)
+#define NORETURN __declspec(noreturn)
+#elif defined(__clang__) || defined(__GNUC__)
+// Use C++11 attribute, supported by Clang and g++ >= 4.8
+#define NORETURN [[noreturn]]
+#else
+#define NORETURN
+#endif
```

Comment:
Why care about MSVS if it doesn't support C++11? It probably won't compile anyway, with or without this macros.